spec/support/vcr_cassettes/FastTrack_DisabilityCompensationJob/_perform/success/the_claim_IS_for_hypertension/emails_the_stakeholders.yml

```diff
@@ -0,0 +1,264 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://sandbox-api.va.gov/oauth2/health/system/v1/token
+    body:
+      encoding: US-ASCII
+      string: grant_type=client_credentials&client_assertion_type=urn%3Aietf%3Aparams%3Aoauth%3Aclient-assertion-type%3Ajwt-bearer&client_assertion=eyJhbGciOiJSUzI1NiJ9.eyJpc3MiOiIwb2FheTM4a3I4TFFicU5XUTJwNyIsInN1YiI6IjBvYWF5MzhrcjhMUWJxTldRMnA3IiwiYXVkIjoiaHR0cHM6Ly9kZXB0dmEtZXZhbC5va3RhLmNvbS9vYXV0aDIvYXVzOG5tMXEwZjdWUTBhNDgycDcvdjEvdG9rZW4iLCJleHAiOjE2Mzg5MDc3MTR9.WSO7ubkV6vjHFrV5_j8Fr0KYiczZmcZu5l1cuKNM_rs6xfrAP9-Dl-RWFJj_6mPL3JAVYz_X9uDGUdfwywMVLuNSJ9ClZgSXdMNUuyLIlnb33Agjv-kB80TDPMV8xlEoVYifs8NfSEt8vcelf3R92zM2YC3fT1Y3sAHs9RqeaS7kMYA5UN2tlkzvf7jtO-p8o2t3Too_eHKjIvgqq-TOvkelb5wEIcj2AJimayPqB_zmMq537wSSC_hZkKnE8GftHM7lRcl3pwqchCmi_5WD6XEbSF2bbXcC6c80fpNHja94Uf8pPsa_97PWyvqd-0weM0FSExJStNHFGJ4QrJ2nQGrMpZ2OSXOb9O1jjKTh7YT58ck1mEGuIPrz-hOdZsIyn9Fjq6HyaOAKhGMdeAVYserdOUkXSeQUxOqOWJOJWlbqpU1V2Csebe8cpxfKoXe6aryUzM6Ehlo72JurQexPSZfumGbfHFbft4ZvjSC7Hfzf_DnFUKDUQljFdBPbcjPaazc6q8M_rUAEhM8LjpON0QCmf2W48UnamrWQ1V5dAXi7EIH5YcPCnP8QoDiNnBgk0sP0GU_u3o4Wsn3Otz-zDDQlx6AA6Noc10BUHluP4vtHdFRRiveq_l78JO6wATS-_m_9LPfTFMUwbIjZvg-RqRX0C6Nl-D_ZBzRETboezK0&scope=launch+launch%2Fpatient+patient%2FPatient.read+patient%2FCondition.read+patient%2FObservation.read+patient%2FPractitioner.read+patient%2FMedication.read+patient%2FMedicationRequest.read+system%2FEndpoint.read&launch=2000163
```

Comment:
This is also minor, but I remember updating the cassettes after changing the way we set up the JWT token request, so this is less up-to-date in that sense.